src/overlays/actors/ovl_En_Heishi1/z_en_heishi1.c

```diff
@@ -21,37 +43,467 @@ const ActorInit En_Heishi1_InitVars = {
     (ActorFunc)EnHeishi1_Update,
     (ActorFunc)EnHeishi1_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Heishi1/EnHeishi1_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Heishi1/EnHeishi1_Destroy.s")
+f32 sAnimParamsInit[][8] = {
+    { 1.0f, -10.0f, 3.0f, 0.5f, 1000.0f, 200.0f, 0.3f, 1000.0f },
+    { 3.0f, -3.0f, 6.0f, 0.8f, 2000.0f, 400.0f, 0.5f, 2000.0f },
+    { 1.0f, -10.0f, 3.0f, 0.5f, 1000.0f, 200.0f, 0.3f, 1000.0f },
+    { 3.0f, -3.0f, 6.0f, 0.8f, 2000.0f, 400.0f, 0.5f, 2000.0f },
+};
+
+s16 sBaseHeadTimers[] = { 20, 10, 20, 10, 13, 0 };
+
+Vec3f sRupeePositions[] = {
+    { 0.0f, 0.0f, 90.0f },  { -55.0f, 0.0f, 90.0f }, { -55.0f, 0.0f, 30.0f }, { -55.0f, 0.0f, -30.0f },
+    { 0.0f, 0.0f, -30.0f }, { 55.0f, 0.0f, -30.0f }, { 55.0f, 0.0f, 30.0f },  { 55.0f, 0.0f, 90.0f },
+};
+
+s32 sCamDataIdxs[] = {
+    7, 7, 2, 2, 2, 2, 3, 3, 4, 4, 5, 6, 4, 4, 5, 6,
+};
+
+s16 sWaypoints[] = { 0, 4, 1, 5, 2, 6, 3, 7 };
+
+extern AnimationHeader D_06005880;
+extern AnimationHeader D_06005C30;
+extern SkeletonHeader D_0600BAC8;
+
+void EnHeishi1_Init(Actor* thisx, GlobalContext* globalCtx) {
+    s32 pad;
+    EnHeishi1* this = THIS;
+    Vec3f rupeePos;
+    s32 i;
+    u16 time;
+
+    Actor_SetScale(&this->actor, 0.01f);
+    SkelAnime_Init(globalCtx, &this->skelAnime, &D_0600BAC8, &D_06005C30, &this->limbDrawTable,
```

Comment:
Because limbDrawTable and transitionDrawTable are arrays, the & is not necessary.